docs/keys.md

```diff
@@ -1,10 +1,10 @@
 # Keys
 
 > NOTE: This is not a lookup table of key objects provided by KMK. That listing
-> can be found in `keycodes.md`, though that file is not always kept up to date.
-> It's probably worth a look at the raw source if you're stumped: `kmk/keys.py`.
+> can be found in `keycodes.md`. It's probably worth a look at the raw source if
+> you're stumped: `kmk/keys.py`.
 
-This is a bunch of documentation about how physical keypresses translate to
+This is a bunch of documentation about how physical keypress translate to
```

Comment:
This sentence now doesn't flow. Either `about how physical keypress*es* translate` or `about how *a* physical keypress translate*s*`

Reply:
Derp. Fixed a spelling error, screwed grammar. Solved.